### DISCUSSION TITLE
How do I add members to a group

### DISCUSSION THREAD

**Discussion by jiaxinonly** [python-gitlab/python-gitlab#2302] [Q&A]
I have a group that can add users by user_id. How do I add a group?

add users
```python
group.members.create({'user_id': 135, 'access_level': gitlab.const.AccessLevel.DEVELOPER})
```
add group? ()
```python
group.members.create({'group_id': 135, 'access_level': gitlab.const.AccessLevel.DEVELOPER})
```
error: 
```
AttributeError: Missing attributes: user_id
```

**Comment by nejch** [comment]
You don't _add_ a group as a member, you _share_ the group with the other group:
https://docs.gitlab.com/ee/api/groups.html#share-groups-with-groups

See example listed here:
https://python-gitlab.readthedocs.io/en/stable/gl_objects/groups.html#examples

```python
# Share/unshare the group with a group:

group.share(group2.id, gitlab.const.AccessLevel.DEVELOPER)
group.unshare(group2.id)

```
